ajax/libs/css-doodle/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "css-doodle",
+  "filename": "css-doodle.min.js",
+  "description": "A web component for drawing patterns with css.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/yuanchuan/css-doodle.git"
+  },
+  "keywords": [
+    "css",
+    "web-components"
+  ],
+  "author": "yuanchuan",
+  "license": "MIT",
+  "npmName": "css-doodle",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.js"
```

Comment:
Let's use `css-doodle*`, this will be more clear that what files need to be added. Also, this can be prevented the files which we don't need, such as webpack.js. On the other hand, this pattern is flexible to get the css-doodle.css which can be added if the lib release the file in future.
Many thanks.

PS. In fact, the pattern `*.js` is seldom be used. We prefer to use more clear pattern to get the files. (I didn't use this pattern before.)